Assets/HoloToolkit/Input/Scripts/GazeStabilizer.cs

```diff
@@ -136,18 +136,18 @@ private void UpdateInstability(out float positionInstability, out float directio
                 return;
             }
 
-            mostRecentSample = stabilitySamples.ElementAt(stabilitySamples.Count - 1);
+            mostRecentSample = stabilitySamples.Last.Value;
 
-            // All but most recent.
-            for (int i = 0; i < stabilitySamples.Count - 1; ++i)
-            {
+            int i = 0;
+            foreach(GazeSample sample in stabilitySamples)
+            {    
                 // Calculate difference between current sample and most recent sample.
-                positionDelta = Vector3.Magnitude(stabilitySamples.ElementAt(i).position - mostRecentSample.position);
+                positionDelta = Vector3.Magnitude(sample.position - mostRecentSample.position);
 
-                directionDelta = Vector3.Angle(stabilitySamples.ElementAt(i).direction, mostRecentSample.direction) * Mathf.Deg2Rad;
+                directionDelta = Vector3.Angle(sample.direction, mostRecentSample.direction) * Mathf.Deg2Rad;
 
                 // Initialize max and min on first sample.
-                if (i == 0)
+                if (i==0)
                 {
```

Comment:
I believe this can now be replaced with a bool instead of the int as it no longer used?

bool first = true;

And then inside the loop:
if(first)
{
    first = false;
    // Perform actions
}